src/treeChooser/treeChooser.controller.js

```diff
@@ -347,6 +347,7 @@ function TreeChooserController(
         vm.deselectChildren(item);
       }
     }
+    $scope.focusInputDiv();
```

Comment:
Why do we need to focus the input div when something is selected? Won't this mess up multiselect behavior?

Reply:
We can’t focus on the input, since it is not displayed if something is selected.
If we don’t focus, then after selection, the focus of tab is moved to the first block or nowhere.